vllm/compilation/compiler_interface.py

```diff
@@ -547,9 +551,9 @@ def metrics_context(self) -> contextlib.AbstractContextManager:
             return contextlib.nullcontext()
 
 
-def set_inductor_config(config, runtime_shape):
-    if isinstance(runtime_shape, int):
-        # for a specific batchsize, tuning triton kernel parameters
+def set_inductor_config(config, compile_range):
+    if isinstance(compile_range, tuple):
+        # for a specific range of batchsizes, tuning triton kernel parameters
         # can be beneficial
         config["max_autotune"] = True
```

Comment:
Can you add a `TODO(luka): max autotune only present with -O3, and this should live in config: https://github.com/vllm-project/vllm/issues/20283`